lib/cc/engine/flog.rb

```diff
@@ -128,24 +128,27 @@ def issue name, datum, location, score
           return
         end
 
-        remediation_points =
-          (BASE_REMEDIATION_POINTS + (OVERAGE_REMEDIATION_POINTS * score)).
-            round
-
         {
          "type"        => "issue",
          "check_name"  => "Flog Score",
          "description" => datum,
          "categories"  => ["Complexity"],
          "content"     => { "body" => CONTENT },
-         "remediation_points" => remediation_points,
+         "remediation_points" => (remediation_points score),
```

Comment:
hah. While many of us are also lispers... "Seattle-style" just means not using any extra syntax except to clarify ambiguities. In this case, your code is fine, but I'd just have written it in the normal ruby manner `remediation_points(score)`.